kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -0,0 +1,185 @@
+logger = log.get_logger(__name__)
+
+class LocalNvRamVmExtensions(object):
+    def __init__(self):
+        self.nvram_volume = None
+        self.vm_uuid = ''
+
+    def prepare_nvram_before_vm_start(self):
+        nvram_install_path = self.nvram_volume.installPath  # type: str
+
+        if self.vm_uuid == '':
+            raise Exception("invalid vm_uuid: %s" % self.vm_uuid)
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)